iModelJsNodeAddon/api_package/ts/src/test/DgnDb.test.ts

```diff
@@ -12,11 +12,84 @@ import * as path from "path";
 import { openDgnDb } from ".";
 import { IModelJsNative, SchemaWriteStatus } from "../NativeLibrary";
 import { copyFile, dbFileName, getAssetsDir, getOutputDir, iModelJsNative } from "./utils";
+import { Suite } from "mocha";
 
 // Crash reporting on linux is gated by the presence of this env variable.
 if (os.platform() === "linux")
   process.env.LINUX_MINIDUMP_ENABLED = "yes";
 
+describe.skip("performance tests", function (this: Suite) {

```

Comment:
we should have some proper tests and not these local only perf test

Reply:
Yes we should have that but not in this repo.